product_pricelist_direct_print/models/product_category.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = "product.category"
+
+    translatable_complete_name = fields.Char(
+        'Translatable Complete Name', compute='_compute_translatable_complete_name', recursive=True, store=False, translate=True)
+
+    @api.depends('name', 'parent_id.translatable_complete_name')
+    def _compute_translatable_complete_name(self):
+        for category in self:
+            if category.parent_id:
+                category.translatable_complete_name = '%s / %s' % (category.parent_id.translatable_complete_name, self.name)
+            else:
+                category.translatable_complete_name = self.name
```

Comment:
is it necessary? why not simply get value from complete_name

Reply:
-  I have tried again. If I get value from complete_name, it doesn't translate
![image](https://github.com/chaule97/product-attribute/assets/32189890/093b0b0c-3387-4c54-b505-0a2d8afa6042)
-  But if I keep it the code before, it translated
![image](https://github.com/chaule97/product-attribute/assets/32189890/7d29d7e2-116e-4f88-ae37-346ed7e69877)